hopsworks-web/yo/app/views/storageConnectorViewInfo.html

```diff
@@ -143,6 +143,25 @@ <h2 class="modal-title">Storage Connector Details: {{storageConnectorViewInfoCtr
                     </td>
                 </tr>
 
+
+                <tr class="fs-table" ng-if="!storageConnectorViewInfoCtrl.settings.s3IAMRole">
+                    <td style="font-weight:700;" class="fs-table">
+                        Server Encryption Algorithm
+                    </td>
+                    <td class="fs-table">
+                        {{storageConnectorViewInfoCtrl.storageConnector.serverEncryptionAlgorithm}}
+                    </td>
+                </tr>
+
+                <tr class="fs-table" ng-if="!storageConnectorViewInfoCtrl.settings.s3IAMRole">
+                    <td style="font-weight:700;" class="fs-table">
+                        Secret Encryption Key
+                    </td>
+                    <td class="fs-table">
+                        {{storageConnectorViewInfoCtrl.storageConnector.serverEncryptionKey}}
```

Comment:
remove this check. encryption information should be shown regardless of the iam role